api/pkg/transformer/pipeline/util.go

```diff
@@ -68,8 +72,12 @@ func subsetSeriesFromExpression(env *Environment, expression string, subsetIdx *
 func getVal(env *Environment, expression string) (interface{}, error) {
 	cplExpr := env.CompiledExpression(expression)
 	if cplExpr == nil {
-		return nil, fmt.Errorf("compiled expression %s not found", expression)
+		return nil, mErrors.NewInvalidInputErrorf("compiled expression %s not found", expression)
```

Comment:
I think it's not invalid input error since the compilation of expression is completely handled by us.

Reply:
Same suggestion as the one in evalJSONPath. https://github.com/gojek/merlin/pull/320/files#r1022286622